src/Directory.Build.targets

```diff
@@ -105,7 +105,7 @@
     </ItemGroup>
   </Target>
   
-  <Target Name="GetNuGetPackageVersionEx" BeforeTargets="GenerateNuspec" DependsOnTargets="GetBuildVersion">
+  <Target Name="GetNuGetPackageVersionEx" BeforeTargets="GenerateNuspec" AfterTargets="GetBuildVersion">
```

Comment:
I think I'd rather see `BeforeTargets="GenerateNuspec" AfterTargets="GetBuildVersion" DependsOnTargets="GetBuildVersion"` but this is ok

Reply:
Yup, you were right! What I had didn't work for a few cases. And I ended up with what you have here :)